dashboard/app/models/plc/learning_module.rb

```diff
@@ -46,8 +46,4 @@ class Plc::LearningModule < ActiveRecord::Base
   def required?
     module_type == REQUIRED_MODULE
   end
-
-  def name_with_required_tag
```

Comment:
Found this dead code when I was looking at how flex category impacts this model so I deleted it